docs/manual/templates.rst

```diff
@@ -476,6 +476,17 @@ alle gegevens uit het formulier en de waarden ingevuld door de gebruiker.
 
    See :ref:`example_form_with_geometry` for a more detailed example.
 
+Voor formulieren die een betaling vereisen, is het ook mogelijk om informatie over de betaling toe te voegen.
+Als de gebruiker betaalt, kan de status van de betaling in de Object API bijgewerkt worden. De structuur van het
+veld ``payment`` binnen het ``record`` veld is ook per formulier instelbaar met een sjabloon.
+In dit sjabloon kunnen alleen de inzending variabelen (``variables.<naam van variabele>``) en de ``payment`` variabele
```

Comment:
```suggestion
In dit sjabloon kunnen alleen de inzendingsvariabelen (``variables.<naam van variabele>``) en de ``payment`` variabele
```

Reply:
One day I will get this merging of words right... :stuck_out_tongue: